addons/website_event_exhibitor/static/src/scss/event_templates_sponsor.scss

```diff
@@ -97,3 +97,27 @@
         opacity:.1;
     }
 }
+
+
+.dropdown-on-hover {
```

Comment:
I changed the names of the css classes to respect the Odoo's conventions. I don't think that it a good idea to add the class `position-relative` to the dom as this rule is mandatory and only makes sens because of the other rules. It's not really something that can be changed to create variants. By putting the rules in the file, we will not have to copy-past the class everywhere if we reuse the component somewhere else. I also changed the background color of the tooltip to be the same as the body of the page. That way, it will better fit the user's theme.